dts/bindings/charger/sbs,sbs-charger.yaml

```diff
@@ -0,0 +1,5 @@
+description: SBS 1.1 compliant charger (http://www.sbs-forum.org/specs)
+
+compatible: "sbs,sbs-charger"
```

Comment:
question: is `sbs` a vendor?

Reply:
It was a consortium of vendors, but has since been dissolved and folded into other consortium as far as I can tell. SBS compliant chargers and (more importantly) gauges are still around.